src/js/experiments/signUpCTA.js

```diff
@@ -0,0 +1,15 @@
+// https://app.optimizely.com/v2/projects/16812830475/experiments/21229840508/variations
+window.OptimizelyClient.getVariationName({
+  experimentKey: 'dd_update_signup_cta_test',
+  groupExperimentName: 'q1_fy23_docs_disco_experiment_group_test',
+  experimentContainer: '.global-nav--footer',
+  guestExperiment: true,
```

Comment:
`guestExperiment` is false by default. If we are running an experiment on an guest/not signed up user we have to add in this option and set it to true.